index.html

```diff
@@ -1024,6 +1024,17 @@ <h2>The <dfn>InteractionOutput</dfn> interface</h2>
           [=Resolve=] |promise| with <a>[[\value]]</a>.
         </li>
       </ol>
+      <p>
+        While the {{value()}} function provides built-in validation, we recognize that some
+        use cases may require returning values without validation. In such cases, developers
+        can use alternative patterns, such as directly accessing the underlying data 
+        using streams or the {{InteractionOutput/arrayBuffer()}} function (See [[[#validation-arraybuffer-example]]] and [[[#stream-example]]]).
```

Comment:
```suggestion
        using streams or the {{InteractionOutput/arrayBuffer()}} function (see [[[#validation-arraybuffer-example]]] and [[[#stream-example]]]).
```

minor: lowercase see